--- markdown.vim.old	2015-03-23 22:44:25.964776514 +0100
+++ markdown.vim	2015-03-23 22:44:37.312893861 +0100
@@ -71,6 +71,7 @@
 syntax match  mkdLineBreak /  \+$/
 syntax region mkdCode       start=/\\\@<!`/     end=/\\\@<!`/
 syntax region mkdCode       start=/\s*``[^`]*/  end=/[^`]*``\s*/
+syntax region mkdCode       start=/\s*```[^`]*/  end=/[^`]*```\s*/
 syntax region mkdBlockquote start=/^\s*>/       end=/$/           contains=mkdLineBreak,mkdLineContinue,@Spell
 syntax region mkdCode       start="<pre[^>]*>"  end="</pre>"
 syntax region mkdCode       start="<code[^>]*>" end="</code>"
